owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/drawer/DrawerViewModel.kt

```diff
@@ -62,13 +66,39 @@ class DrawerViewModel(
         return AccountUtils.getCurrentOwnCloudAccount(context)
     }
 
+    fun getUsernameOfAccount(accountName: String): String {
+        return AccountUtils.getUsernameOfAccount(accountName)
+    }
+
     fun setCurrentAccount(context: Context, accountName: String): Boolean {
         return AccountUtils.setCurrentOwnCloudAccount(context, accountName)
     }
 
-    fun deleteUnusedUserDirs(accounts: Array<Account>) {
+    fun removeAccount(context: Context) {
```

Comment:
Not a good idea to send the context to the viewModel. Send the accounts instead if we use the context only to retrieve the accounts